.circleci/config.yml

```diff
@@ -57,6 +70,21 @@ jobs:
           paths:
             - docs/docs/_build/html/
 workflows:
+  # default trigger on push, don't double up with actions
```

Comment:
This comment is confusing. It's unclear what "don't double up with actions" means.

Reply:
Thanks, I will try to rephrase.